tests/suites/test_suite_x509parse.function

```diff
@@ -9,6 +9,7 @@
 #include "mbedtls/base64.h"
 #include "mbedtls/error.h"
 #include "string.h"
+#include "pk_wrap.h"
```

Comment:
I was going to ask why `pk_wrap.h` as opposed to `pk.h` and the answer is: because that's where `ECDSA_SOME` is defined. Which made me think, it would probably be better for `ECDSA_SOME` to be defined in the public header `pk.h` as well: if we need it, chances are users may need it too, so it's probably better to put it in the public header. Also, it seems more consistent to have all the `PK_CAN` macros in the same place.

Then this and other places which include `pk_wrap.h` only for this macro can be changed to include `pk.h` instead.

(The header `pk_wrap.h` is supposed to be about internal functions and structures for the implementation of PK, so it's a minor smell to see it included from tests and other library modules such as `oid.c`